### PR TITLE
Futureevents

### PR DESCRIPTION
**Pull request by shua123**
The main focus of this pull request are updates to handle (past and) future events:

1. On the homepage, below the Next Event box, there is a Last Event box to highlight the most recent event. If there is a video associated with the event, it will link to the video. https://github.com/chihacknight/chihacknight.org/blob/futureevents/index.html#L62
image
2. Below that, there is a Future Events box that shows the remaining future events. https://github.com/chihacknight/chihacknight.org/blob/futureevents/index.html#L97
image

Additionally, I moved the previously commented out holiday "week off" sections to includes to reduce some of the index.html clutter. https://github.com/chihacknight/chihacknight.org/commit/cbb51f01466a21c7f772c8dd768b08c4096e8504

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,16 +45,70 @@ <h2 class='no-margin-top'>
...
+    </div>
+      
+  </div>
+</div>
+{% endfor %}
+
+{% assign future_events = upcoming_posts | shift %}
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title"><b>Future Events</b> <a class='pull-right' href='/events/'>Past events &raquo;</a></h3>
```
Comment (derekeder) [inline index.html]: might call this "Upcoming Events"

File: index.html [hunk 2]
```diff
@@ -95,16 +45,70 @@ <h2 class='no-margin-top'>
...
+      <div class='col-sm-12'>
+        {% if post.description %}
+          <p>{{ post.description }}</p>
+        {% endif %}
+        <a class='btn btn-link btn-lg' href="{{post.url}}"><i class='fa fa-info-circle-square-o'></i> Details</a>
+        {% if post.youtube_id %}
+        <a class='btn btn-link btn-lg' href="https://www.youtube.com/watch?v={{post.youtube_id}}" target='_blank'><i class='fa fa-play-circle'></i> YouTube Video</a>
+        {% endif %}
+      </div>
+      <!-- <div class='col-sm-5'>
```
Comment (derekeder) [inline index.html]: should we delete this commented section?

File: lib/events.js [hunk 3]
```diff
@@ -9,7 +9,8 @@ var events_table = $('#hack-night-events').dataTable( {
         null
     ],
     "bInfo": false,
-    "bPaginate": false
+    "bPaginate": true,
```
Comment (derekeder) [inline lib/events.js:12]: I like the pagination - 450+ entries is getting way too long for one page.

However the style of the pagination controls is a little funky. We should use the CHN blue/red colors:

Screen Shot 2021-09-02 at 8 49 23 AM

**Review by derekeder — CHANGES_REQUESTED** [review]
@shua123 this is looking really good! I made a few small comments - the biggest being the style of the pagination controls.

I'd like @easherma's input/review too.

File: index.html (hunk 1 above)
Comment (shua123) [reply index.html]: Good idea.

File: lib/events.js (hunk 3 above)
Comment (shua123) [reply lib/events.js:12]: I switched these to CHN colors

File: index.html (hunk 2 above)
Comment (shua123) [reply index.html]: Done https://github.com/chihacknight/chihacknight.org/pull/389/commits/6ffeb4f38c55805f9f36f21e2fe4dfc10a7b1b87

**Review by derekeder — APPROVED** [review]
@shua123 this is looking good to me. I'm ok with bringing this in. 

@easherma what say you?

**Comment by derekeder** [comment]
@shua123 what do you think - should we just merge this in?

**Comment by shua123** [comment]
Yeah. I'll work on that this weekend.

**Comment by easherma-truth** [comment]
I haven't tested this, but skimming the code and it looks like some useful improvements!

**Pull request merged**